plugins/msr/fu-plugin-msr.c

```diff
@@ -369,6 +372,7 @@ fu_plugin_add_security_attr_amd_sme_enabled(FuPlugin *plugin, FuSecurityAttrs *a
 	if (!(fu_plugin_msr_kernel_enabled_sme(&error_local))) {
 		g_debug("%s", error_local->message);
 		fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_NOT_ENCRYPTED);
+		fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_ACTION_CONFIG_FW);
```

Comment:
Should be contact vendor - they decide whether or not to offer SME.  TSME is changable in firmware

Reply:
Done.